controller/src/beerocks/master/son_master_thread.cpp

```diff
@@ -1552,24 +1555,29 @@ bool master_thread::autoconfig_wsc_parse_radio_caps(
         auto &op_class                  = std::get<1>(operating_class_tuple);
         auto operating_class            = op_class.operating_class();
         auto maximum_transmit_power_dbm = op_class.maximum_transmit_power_dbm();
-        ss << "operating_class=" << int(operating_class);
-        ss << "maximum_transmit_power_dbm=" << int(maximum_transmit_power_dbm);
-        ss << ", statically_non_operable_channel_list={";
+        ss << "operating_class=" << int(operating_class) << std::endl;
+        ss << "maximum_transmit_power_dbm=" << int(maximum_transmit_power_dbm) << std::endl;
+        ss << "channel list={ ";
+        auto channel_list = son::wireless_utils::operating_class_to_channel_set(operating_class);
```

Comment:
auto&

Reply:
auto& where?

Reply:
auto& channel_list

Reply:
`operating_class_to_channel_set` doesn't return reference (and it shouldn't)